packages/element/src/create-interpolate-element.js

```diff
@@ -0,0 +1,330 @@
+/**
+ * External dependencies
+ */
+import { createElement, cloneElement, Fragment, isValidElement } from 'react';
+
+let indoc,
+	offset,
+	output,
+	stack;
+
+/**
+ * Matches tags in the localized string
+ *
+ * This is used for extracting the tag pattern groups for parsing the localized
+ * string and along with the map converting it to a react element.
+ *
+ * There are four references extracted using this tokenizer:
+ *
+ * match: Full match of the tag (i.e. <strong>, </strong>, <br/>)
+ * isClosing: The closing slash, it it exists.
+ * name: The name portion of the tag (strong, br) (if )
+ * isSelfClosed: The slash on a self closing tag, if it exists.
+ *
+ * @type RegExp
+ */
+const tokenizer = /<(\/)?(\w+)\s*(\/)?>/g;
+
+/**
+ * An object describing a component to be created.
+ *
+ * This is used by the string iterator to track children that get added to an
+ * element when it is created. This allows for collecting nested elements in
+ * the string before creating the parent.
+ *
+ * @private
+ * @param {string}    name    The name of the component.
+ * @param {WPElement} element The element
+ *
+ * @return {Component} An object returning the creator and children.
+ */
+function Component( name, element ) {
+	return {
+		name,
+		element,
+		children: [],
+	};
+}
+
+/**
+ * Tracks recursive-descent parse state.
+ *
+ * This is a Stack element holding parent elements until all children have been
+ * parsed.
+ *
+ * @private
+ * @param {Component} component        A parent element which may still have
+ *                                     nested children not yet parsed.
+ * @param {number}    tokenStart       Offset at which parent element first
+ *                                     appears.
+ * @param {number}    tokenLength      Length of string marking start of parent
+ *                                     element.
+ * @param {number}    prevOffset       Running offset at which parsing should
+ *                                     continue.
+ * @param {number}    leadingTextStart Offset at which last closing element
+ *                                     finished, used for finding text between
+ *                                     elements
+ *
+ * @return {Frame} The stack frame tracking parse progress.
+ */
+function Frame(
+	component,
+	tokenStart,
+	tokenLength,
+	prevOffset,
+	leadingTextStart
+) {
+	return {
+		component,
+		tokenStart,
+		tokenLength,
+		prevOffset: prevOffset || tokenStart + tokenLength,
```

Comment:
In lookly closely, I don't think this expression is needed, it can just be `prevOffset`.  I think there may have been an iteration where it could have been something different.

Reply:
see ec9afec